app/views/categories/show.html.erb

```diff
@@ -5,5 +5,5 @@
 <p><%= link_to t(".navigation.helpers.back"), categories_path %>
 
 <% posts.each do |post| %>
-  <%= render "posts/blog_post", post: post %>
+  <%= render "posts/post", post: post %>
 <% end %>
```

Comment:
I think you can replace this entire loop with:

```ruby
<%= render posts %>
```

Reply:
I wasn't able to get this to work because it's on the `category/show` page.  It went looking for a template under `category/_posts`.  Am I missing something Rails-y?

Reply:
Oh interesting. I checked the documentation for that before I said it and it seemed to imply that it would look at `posts/_post` (https://guides.rubyonrails.org/layouts_and_rendering.html#using-partials).

According to this thoughtbot blog post though (https://thoughtbot.com/blog/rendering-collections-in-rails) it looks like what you're seeing is expected, and it is possible to override the `partial_path` method on the model (although it probably isn't worth it!).

Reply:
Wow, that's very interesting!  Think I'll stick with my ugly code for now until it bothers me enough to change it, but this will likely come in handy for future client work :)